orga/app/components/password-reset-modal.js

```diff
@@ -59,6 +60,25 @@ export default class PasswordResetModal extends Component {
     }
   }
 
+  @action
+  async generateUsernameWithTemporaryPassword(event) {
+    event.preventDefault();
+    const schoolingRegistrationDependentUser = this.store.createRecord('schooling-registration-dependent-user', {
+      organizationId: this.args.organizationId,
+      schoolingRegistrationId: this.args.student.id
+    });
+
+    try {
+      await schoolingRegistrationDependentUser.save({ adapterOptions: { generateUsernameAndTemporaryPassword: true } });
+      this.args.student.username = schoolingRegistrationDependentUser.username;
+      this.generatedPassword = schoolingRegistrationDependentUser.generatedPassword;
+      this.isUniquePasswordVisible = !this.isUniquePasswordVisible;
+    } catch (response) {
+      const errorDetail = get(response, 'errors[0].detail', 'Une erreur est survenue, veuillez réessayer ultérieurement.');
```

Comment:
```suggestion
      const errorDetail = get(response, 'errors[0].detail', 'Une erreur interne est survenue, nos équipes sont en train de résoudre le problème. Veuillez réessayer ultérieurement.');
```

Si possible de le mettre dans une constante globale à l'image de Pix App pour que ce message soit réutilisable et le même partout.

Reply:
Done

J'ai juste créé une constante dans la class PasswordResetModal.
Dans Pix App, la gestion de la locale est ON, et les messages d'erreurs API sont gérées selon le statusCode HTTP.
Ce n'est pas encore le cas dans Pix Orga.